client/src/modules/dashboard/index.jsx

```diff
@@ -16,13 +15,13 @@ import { getCohorts } from 'modules/cohort/model/selectors';
 import { getCurrentUser } from 'modules/authorization/model/selectors';
 import { UserPropType } from 'common/constants/propTypes';
 import GridList from 'common/components/GridList';
-import DropdownForm from 'common/components/DropdownForm';
 import { CardColorType } from 'common/components/CardItem';
+import FormDialog, { FormDialogContext } from 'common/components/FormDialog';
 
 class Dashboard extends Component {
   state = {
-    cohortFormVisibility: false,
-    listFormVisibility: false
+    formDialogVisibility: false,
```

Comment:
remove and use only `formDialogContext` to determine if dialog should be shown or hidden